app/presenters/applications_index_presenter.rb

```diff
@@ -125,7 +125,7 @@ def raw_applications
   def applications
     @applications ||= raw_applications.scope_search(search)
                                       .order_by(*sorting_params)
-                                      .preload(:service, user_account: %i[admin_user], plan: %i[pricing_rules])
+                                      .includes(plan: %i[pricing_rules])
```

Comment:
Sorry for my laziness - what is the main difference between `preload` and `includes` ? :grimacing:

Reply:
AFAIU `#preload` simply preloads what was requested with separate queries. While `#includes` is supposed to be smarter and extract objects from joins as well. Also it accumulates with other includes. There is a caveat that if in the join some associated objects are filtered, they are not seen by `#includes`.

In this case I don't want to mix them because IIRC there was some `includes` used in a chain and probably that will play better together with it. I'm not sure whether `preload` plays together well with `includes`. But also see no reason to experiment. `includes` seems to be the primary documented way. Although some people are concerned about possible surprises with it due to filtering of related objects in joins.

Reply:
Thanks a lot for the explanation!